src/components/header/Header.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import styled from 'styled-components';
+import HeaderIcon from './HeaderIcon';
+import NavBar from './NavBar';
+
+const HeaderDiv = styled.div`
+  width: 100%;
+  height: 100px;
+  display: flex;
+  flex-direction: row;
+  position: fixed;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 80px;
+  z-index: 1;
+  background: white;
+`;
+
+export default function Header() {
+  return (
+    <header>
+      <HeaderDiv>
```

Comment:
You can actually remove the `HeaderDiv` and replace it with a styled header. So above instead of writing `const HeaderDiv = styled.div` you can write 

```
const Header = styled.header`
  width: 100%;
  height: 100px;
  display: flex;
  // and so on
```

So you save one DOM element 😉